setup.py

```diff
@@ -33,6 +33,7 @@ def read(fname):
 
 
 data_files = [('/usr/lib/systemd/system', glob('systemd/*.service')),
+              ('/etc/anaconda/conf.d', glob('data/*.conf')),
```

Comment:
I think this needs to be added to the spec too.

Reply:
Fixed.